syntax = "proto3";

package requestPackage;

option go_package = "./request";

service GradeService {
  rpc GradeFile(File) returns (Status);
}

message File {
  string fileid = 1;
  string filename = 2;
}

message noParams {}

message Status {
  int32 statusCode = 1;
}

